### DISCUSSION TITLE
GENIOS-Hack: Passwörter ändern?

### DISCUSSION THREAD

**Discussion by florisre** [stefanw/voebbot#452] [General]
Angesichts des Hacks der GENIOS-Datenbank folgende Frage: muss ich mein Bibliothekspasswort ändern? Denn die Browsererweiterung übermittelt ja meine Zugangsdaten an die Server von GENIOS, die nicht länger vertrauenswürdig sind.

**Comment by stefanw** [comment]
Grundsätzlich kann ich dazu keine definitive Antwort geben. BibBot gibt das Passwort in das Passwortfeld von Genios ein, so wie Sie es ohne BibBot selber auch tun würden. Wenn das Eingabefeld nicht da ist, wird kein Passwort übermittelt. Ob der "Hackerangriff" die Passwörter aber schon vor Abschalten der Loginseite abgegriffen hat oder irgendeine andere Art der Kompromittierung von Zugängen stattgefunden hat, kann nur Genios wissen. Ich würde dort oder bei Ihrer Bibliothek nachfragen.
